Slightly Clarify Race Condition Docs (#4030)
braze-inc/braze-docs: _docs/_user_guide/engagement_tools/testing/race_conditions.md

## Using multiple API endpoints

> If you're using separate API endpoints to create users and trigger Canvases/campaigns, this can also result in this race condition. When user information is sent to Braze via the `users/track` endpoint, it may occasionally take a few seconds to process. As a result, when requests are made to the `users/track` and [messaging endpoints][4] at the same time, there is currently no guarantee that the user information will be updated before a message is sent. If these requests are made in the same API request, there should be no issue. Note that if you are sending a scheduled message API request, these requests must be separate, and a user must be created before sending the schdeuled API request.

Reviewer: I'm confused about "these requests"—requests to the messaging endpoints and `users/track` in the same API request? Maybe reword a little more?

{% alert note %}

> If user attributes are sent in the same request as the event (either from `users/track` or from the SDK), then Braze will generally process those first before attempting to send any message.

Reviewer 2: ```suggestion
If user attributes and events are sent in the same request (either from `users/track` or from the SDK), then Braze will generally process attributes before events or attempting to send any message.
```
